runtime/lua/vim/snippet.lua

```diff
@@ -179,13 +177,16 @@ function Tabstop:set_right_gravity(right_gravity)
   })
 end
 
+--- @alias vim.snippet.NavigationMappings { jump_forward: string, jump_backward: string }
```

Comment:
could leave the door open for other kinds of mappings in the future?

also given `vim.keymap` maybe we should stick to "keymap" 

```suggestion
--- @alias vim.snippet.Keymap { jump_forward: string, jump_backward: string }
```